cli/dts/lib.deno.worker.d.ts

```diff
@@ -51,20 +53,24 @@ declare class WorkerGlobalScope extends EventTarget {
   Deno: typeof Deno;
 }
 
+/** @category Web APIs */
 declare class WorkerNavigator {
```

Comment:
wouldnt this make more sense to be categorized as worker?

Reply:
Why? It is simply the same as "Navigator" but scoped to what is available in the worker... it contains just general APIs.